contracts/Logger.sol

```diff
@@ -6,20 +6,14 @@ contract Logger {
     /*********************
      * Rollup Contract *
      ********************/
-    event NewBatch(
-        address committer,
-        bytes32 updatedRoot,
```

Comment:
Removed the updated Root, because we are expected to get
- all submitted state roots from the calldata.
- The genesis root can be accessed through the contructor.

Reply:
Makes sense to me